app/src/main/java/com/github/gotify/init/BootCompletedReceiver.java

```diff
@@ -0,0 +1,28 @@
+package com.github.gotify.init;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import com.github.gotify.Settings;
+import com.github.gotify.service.WebSocketService;
+
+public class BootCompletedReceiver extends BroadcastReceiver {
+
+    private Settings settings;
```

Comment:
This could be a local variable in the onReceive method.

Reply:
Oh yeah, of course. That's a leftover from looking at your InitializationActivity class. Will change it right now.